packages-exp/analytics-compat/rollup.shared.js

```diff
@@ -45,7 +45,7 @@ export const es2017BuildsNoPlugin = [
      */
     input: 'src/index.ts',
     output: {
-      file: pkg.esm2017,
+      file: pkg.browser,
```

Comment:
I can't put comments for the es5 build because it's not part of the PR - `pkg.module` should point to the esm2017 build, and `pkg.esm5` should be used to create the es build.

Reply:
Fixed.